app/views/admin/grove_monitor/_student.html.erb

```diff
@@ -1,10 +1,16 @@
-<div class="col-xs-3">
-  <div class="thumbnail">
-    <img src="..." alt="...">
-    <div class="caption">
-      <h3><%= student.name if student %></h3>
-      <p>...</p>
-      <p><a href="#" class="btn btn-primary" role="button">Button</a> <a href="#" class="btn btn-default" role="button">Button</a></p>
+<% if student %>
+  <div class="col-xs-3 student">
+    <div class="thumbnail">
+      <img src="..." alt="...">
+      <div class="caption">
+        <h3><%= student.name %></h3>
+        <p>...</p>
+        <% if student.at_school %>
+          <%= link_to "Mark as Absent", admin_update_grove_monitor_all_path(id: student.id), method: :patch, class: "btn btn-default" %>
+          <% else %>
```

Comment:
you might think about putting params in this:
`<%= link_to "Mark as Absent", admin_update_grove_monitor_all_path(id: student.id, student: {at_school: false} ), method: :patch, class: "btn btn-default" %>`

likewise for present below just with `at_school: true`